reo/validators.py

```diff
@@ -246,6 +284,26 @@ def recursive_search(item,level=0, entry=0):
             recursive_search(schedule)
         return valid
 
+    def validNumber(self, name):
+        try:
+            float(getattr(self,name, 0))
+        except:
+            self.errors.append('Entry for {} ({}) is not a valid number.'.format(name, getattr(self,name)))
+            return False
+        return True
+
+    def isNotNone(self, name):
+        if getattr(self,name, None) is None:
+            self.errors.append('Missing valid entry for {}.'.format(name))
+            return False
+        return True
+    
+    def isNotEmptyList(self, name):
+        if type(getattr(self,name)) != list:
+            self.errors.append('Expecting a list for {}.'.format(name))
+            if len(getattr(self,name)) == 0:
```

Comment:
should the `if` for the length check be de-dented? (Unless it self.name is a list it probably doesn't have a length).